lcc_members/models/res_partner.py

```diff
@@ -0,0 +1,124 @@
+# Copyright 2020 Lokavaluto ()
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields
+from odoo.tools.translate import _
+
+
+class res_partner(models.Model):
+    _inherit = "res.partner"
+
+    reasonschoosingmlc = fields.Text(
+        string=_("Why did I choos local monay"),
+        required=False,
+        translate=True,
+        readonly=False
+    )
+    instagram = fields.Char(
+        string=_("Instagram"),
+        required=False,
+        translate=False,
+        readonly=False
+    )
+    phone_pro = fields.Char(
+        string=_("Professional phone"),
+        required=False,
+        translate=False,
+        readonly=False
+    )
+    member_comment = fields.Text(
+        string=_("Commentaires membre"),
+        required=False,
+        translate=False,
+        readonly=False
+    )
+    member_type_id = fields.Many2one(
+        string=_("Type de membres"),
+        required=False,
+        translate=False,
+        readonly=False
+    )
+    convention_signature_date = fields.Date(
+        string=_("convention signature date"),
+        required=False,
+        translate=False,
+        readonly=False
+    )
+    to_renew = fields.Boolean(
+        string=_("To renew"),
+        required=False,
+        translate=False,
+        readonly=False
+    )
+    legal_activity_code = fields.Char(
+        string=_("legal activity code (ex. APE in France)"),
+        required=False,
+        translate=False,
+        readonly=False
+    )
+    facebook_url = fields.Char(
+        string=_("URl Facebook"),
+        required=False,
+        translate=False,
+        readonly=False
+    )
+    keywords = fields.Text(
+        string=_("keywords"),
+        required=False,
+        translate=True,
+        readonly=False
+    )
+    currencyexchangeoffice = fields.Boolean(
```

Comment:
```suggestion
    is_currency_exchange_office = fields.Boolean(
```

Reply:
@jcleonard2018 we need to remove this as asked in an other task right ?

Reply:
Yes currency exchange office is managed using partner tags